src/renderer/components/content/editor/editor.js

```diff
@@ -16,20 +15,16 @@ var Editor = React.createClass({
   //Be careful with changing props, can wipe noteblocks if blocks prop is messed with
   //Everything is Asynchronous, and using replaceState will wipe all blocks, deleting the note
 
-  getInitialState: function() {
-    return {
-      changed: false
-    };
-  },
+  changed: false,
 
   componentDidMount: function() {
-    this.props.note.on('all', this.updateComp, this);
+    this.props.note.on('block-updated', this.updateComp, this);
```

Comment:
Can this be changed back to listen to 'all'? We might need to add other note related events to force updates in the future.